src/app/admin/tags/page.tsx

```
@@ -1,237 +1,262 @@
+								<Plus className="size-4" />
+								Add Tag
+							</Button>
+						</DialogTrigger>
+						<DialogContent className="sm:max-w-[500px]">
+							<DialogHeader>
+								<DialogTitle>Create New Tag</DialogTitle>
+								<DialogDescription>Add a new tag to the global tag system.</DialogDescription>
+							</DialogHeader>
+							<div className="grid gap-4 py-4">
+								<div className="grid gap-2">
+									<label className="text-sm font-medium">Tag Name</label>
```

Biome (2.1.2)

[error] A form label must be associated with an input.

Consider adding a `for` or `htmlFor` attribute to the label element or moving the input element to inside the label element.

(lint/a11y/noLabelWithoutControl)